web/styles/compose.css

```diff
@@ -716,11 +717,10 @@ input.recipient_box {
 
 .compose_right_float_container {
     display: flex;
-    flex-direction: column;
-    align-items: flex-end;
+    flex-direction: row;
     white-space: nowrap;
-    gap: 4px;
     margin-top: 2px;
+    height: 24px;
```

Comment:
Do you know why this part of the compose box was set to `flex-direction: column` before? I'm wondering if there's a mobile screen size possible issue to think about here.

Reply:
I recall it was part of my effort to keep `Send` button and `enter to send` in the same `div` element, but didn't happen due to design issues. So, `flex-direction: column` was not being used for any purpose here.